app/models/theme.rb

```diff
@@ -699,6 +699,26 @@ def build_local_theme_uploads_hash
     hash
   end
 
+  # Retrieves a theme setting
+  #
+  # @param setting_name [String, Symbol] The name of the setting to retrieve.
+  #
+  # @return [Object] The value of the setting that matches the provided name.
+  #
+  # @raise [Discourse::NotFound] If no setting is found with the provided name.
+  #
+  # @example
+  #   theme.get_setting("some_boolean") => True
+  #   theme.get_setting("some_string") => "hello"
+  #   theme.get_setting(:some_boolean) => True
+  #   theme.get_setting(:some_string) => "hello"
+  #
+  def get_setting(setting_name)
+    target_setting = settings.find { |setting| setting.name == setting_name.to_sym }
```

Comment:
Why the `.to_sym` in `setting_name.to_sym` when in the `update_setting` method right after we aren't doing it? 🤔

Reply:
O I did this because I'm designing a new API here and didn't want to touch other methods within this PR. I can follow up with that change in a following PR.